tests/api/bloom.js

```diff
@@ -0,0 +1,76 @@
+const tape = require('tape')
+const Bloom = require('../../lib/bloom')
+const utils = require('ethereumjs-util')
+
+const byteSize = 256
+
+tape('bloom', (t) => {
+  t.test('should initialize without params', (st) => {
+    const b = new Bloom()
+    st.deepEqual(b.bitvector, utils.zeros(byteSize), 'should be empty')
+    st.end()
+  })
+
+  t.test('shouldnt initialize with invalid bitvector', (st) => {
+    st.throws(() => new Bloom('invalid'), /AssertionError/, 'should fail for invalid type')
+    st.throws(() => new Bloom(utils.zeros(byteSize / 2), /AssertionError/), 'should fail for invalid length')
+    st.end()
+  })
+
+  t.test('should contain values of hardcoded bitvector', (st) => {
+    const hex = '00000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000010000000000000000000000000000000000000000000000000000000000000000008000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000200000020000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000000'
+    const vector = Buffer.from(hex, 'hex')
+
+    const b = new Bloom(vector)
+    st.true(b.check('value 1'), 'should contain value 1')
+    st.true(b.check('value 2'), 'should contain value 2')
+    st.end()
+  })
+
+  t.test('check shouldnt be tautology', (st) => {
+    const b = new Bloom()
+    st.false(b.check('random value'), 'should not contain random value')
+    st.end()
+  })
```

Comment:
This is already high-level criticism, don't necessarily needs to be updated, but the explanatory strings are bit misleading on first read, would be more clear with something like ``'should not contain string "random value"'``, a bit similar for other test messages.

Again, change or keep as you like - just as a note and maybe a bit picky. 😄

Otherwise: OK.